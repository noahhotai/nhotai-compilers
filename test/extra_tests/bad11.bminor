// Local arrays with non integer expression size
foo: function integer () = {
    c: float = 3.2;
    b: float = 3.2 + c;
    a: array [b] integer;

    d: array [2.1 + 3.5 * 2.6] integer;
}
