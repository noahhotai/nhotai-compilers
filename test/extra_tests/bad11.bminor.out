Program parsed successfully.
c resolves to local 1
Program resolved successfully.
Program type checked successfully.
